app/src/main/res/layout/state_fragment.xml

```diff
@@ -99,57 +99,87 @@
       android:layout_width="match_parent"
       android:layout_height="match_parent" />
 
-    <FrameLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_gravity="bottom|start"
+      android:layout_marginTop="24dp"
       android:clipChildren="false"
       android:clipToPadding="false"
       android:paddingTop="24dp">
-
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/hints_and_solution_fragment_container"
+      <FrameLayout
+        android:id="@+id/quick_switch_exploration_language_button_container"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/component_color_shared_hints_background_color"
-        android:clipChildren="false"
-        android:visibility="@{viewModel.isHintBulbVisible() ? View.VISIBLE : View.GONE}">
-
-        <ImageView
-          android:id="@+id/hint_bulb"
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:layout_margin="8dp"
-          android:contentDescription="@{viewModel.isHintOpenedAndUnRevealed() ? @string/new_hint_available : @string/no_new_hint_available}"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent"
-          app:srcCompat="@{viewModel.isHintOpenedAndUnRevealed() ? @drawable/ic_hint_bulb_filled_yellow_48dp : @drawable/ic_hint_bulb_white_48dp}" />
-
-        <TextView
-          android:id="@+id/hint_bar_title"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/animable_hints_and_solution_fragment_container"
+        android:background="@color/color_palette_transparent_background_color"
```

Comment:
I think this is by mistake but here we can use `component_color_shared_transparent_background_color` in replace of `color_palette_transparent_background_color`



Also in this file please format code by code format shortcut in android studio.